README.md

```diff
@@ -54,47 +83,78 @@ The pattern of usage is likely to depend on whether a [code first or design firs
 ```
 
 #### Parameters
-Parameter | Description | Required | Default
---------- | ----------- | --------- | -------
-**`api`** | API name | true  | -
-**`owner`** | API owner | true | -
-**`version`** | API version | true | -  
-**`outputFile`** | API definition is written to this file | true | - 
-**`token`** | SwaggerHub API key, required to access private definitions | false | -
-**`format`** | API definition format, `json` or `yaml` | false | `json`
-**`host`** | URL of SwaggerHub API | false | `api.swaggerhub.com`
-**`protocol`** | Protocol for SwaggerHub API,`http` or `https` | false | `https`
-**`port`** | Port to access SwaggerHub API| false | `443`
-
-***
+
+Parameter | Description | Required? | Default
+--------- | ----------- | -------- | -------
+**`api`** | API name (case-sensitive) | yes  | -
+**`owner`** | API owner (case-sensitive) | yes | -
+**`version`** | API version (case-sensitive) | yes | -
+**`outputFile`** | API definition will be saved to this file | yes | -
+**`token`** | SwaggerHub API key, required to access private definitions | no | -
+**`format`** | API definition format, `json` or `yaml` | no | `json`
+**`host`**, **`protocol`**, **`port`** | Reserved for future use | no |
```

Comment:
@john-french the plugin is currently tailored for SaaS only. On-Prem API calls require `/v1` after the hostname, and the plugin does not support that. I removed mentions of On-Prem to avoid confusion.

Reply:
Ah ok, my mistake @hkosova. My intent was for it to support on-prem with these settings, and I successfully tested this functionality (or so I thought). What I actually tested against was a local (development) deployment of SwaggerHub SaaS, which, it turns out, is not the same as a deployment of on-prem.